requirements.txt

```diff
@@ -1,18 +1,25 @@
 pandas
 torch
 torchvision
-scikit-learn>=0.22.0,<0.23
+scikit-learn
```

Comment:
I think we should limit out version of scikit learn -- because there are normally very big changes in the latest versions